desc/protoparse/parser_test.go

```diff
@@ -326,3 +326,15 @@ func TestBasicValidation(t *testing.T) {
 		}
 	}
 }
+
+func TestMethodOptions(t *testing.T) {
```

Comment:
Let's rename this to `TestAggregateValueInUninterpretedOptions` since that's really where the problem was.

Reply:
Done.